config/locales/diaspora/en.yml

```diff
@@ -1008,8 +1008,13 @@ en:
 
   profiles:
     edit:
-      your_public_profile: "Your public profile"
-      your_private_profile: "Your private profile"
+      basic: "Basic"
+      additional: "Additional"
```

Comment:
I'd vote for 'My basic profile'/'My extended profile'.

I think it is a good idea to change the word "private" to the word "limited" there.

Currently we have a note near additional profile (see 'additional_hint' below). I could add similiar hint saying "Your basic profile will always be publicly visible" to basic profile section.